build.gradle

```diff
@@ -32,7 +32,6 @@ buildscript {
 
     repositories {
         mavenLocal()
-        jcenter()
```

Comment:
❓  Shouldn't we have mavenCentral here just like we have for plugins?

Reply:
yep, let's add it

Reply:
Still no mavenCentral here :)